### ISSUE TITLE
Failure to find game metadata on IGDB.

### ISSUE THREAD

**Issue by ghost** [JosefNemec/Playnite#1002]
Sometimes the client fails to find any results when searching IGDB when other times its fine for the same game. This problem only occurs with IGDB and never when using the Wiki option.

**Comment by JosefNemec** [comment]
Can you please go to `About` menu and generate diagnostics package for me? Also please provide names of games that you know could reproduce this. Thank you

**Comment by JosefNemec** [comment]
I've made some optimizations to our IGDB API service to hopefully mitigate this, but I won't be able to say for sure that it's fixed unless I have more information about specific games causing this to happen.

**Comment by DennoCoil** [comment]
This is the first time I've found this bug.  The game is called "Noitu Love 2 Devolution"

I did a search on IGDB, and try to download the first request.
![2019-03-01 17_58_11-edit game details](https://user-images.githubusercontent.com/21137176/53671281-b21d7700-3c4b-11e9-8139-0501ccb23d40.png)

The game exists and there's plenty of information about it.
https://www.igdb.com/games/noitu-love-2-devolution
